dts/riscv/starfive/jh7110-visionfive-v2.dtsi

```diff
@@ -133,9 +133,14 @@
 		};
 	};
 
-	ram0: memory@8000000 {
+	l2lim: memory@8000000 {
 		device_type = "memory";
-		reg = <0x0 0x8000000 0x1 0x0>;
+		reg = <0x0 0x8000000 0x0 0x200000>;
+	};
+
+	dtim: memory@01100000 {
```

Comment:
Remove the leading zero in memory address. 
It gives warning during the build process.
```suggestion
	dtim: memory@1100000 {
```

Reply:
Ah yes, that is annoying. I wanted to use 8 digits to make it easier to read. Fix to come!